tests/zfs-tests/tests/functional/trim/setup.ksh

```diff
@@ -24,6 +24,7 @@
 verify_runnable "global"
 
 if is_freebsd; then
+	log_unsupported "FreeBSD has no hole punching mechanism for the time being."
 	diskinfo -v $DISKS | grep -qE 'No.*# TRIM/UNMAP support' &&
```

Comment:
The diskinfo command is dead code now.

Reply:
I left it for when we do enable these tests.